lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -18,13 +18,27 @@ class PokemonOverviewPage extends StatelessWidget {
       appBar: AppBar(title: const Text(pokedexTitle)),
       body: pokemons.when(
         loading: () => const Center(child: CircularProgressIndicator()),
-        error: (errorMessage) => Center(
-          child: Text(
-            errorMessage ?? emptyString,
-            style: const TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
-            textAlign: TextAlign.center,
-          ),
-        ),
+        error: (errorMessage) {
+          final snackBar = SnackBar(
+            content: Text(
+              errorMessage ?? emptyString,
+              style: const TextStyle(color: Colors.red),
+            ),
+          );
+          WidgetsBinding.instance.addPostFrameCallback((_) {
+            ScaffoldMessenger.of(context).showSnackBar(snackBar);
+          });
+          return const Center(
+            child: Text(
+              pokemonOverviewPageErrorMessage ?? emptyString,

```

Comment:
pokemonOverviewPageErrorMessage constant has no way it's gonna be null right? therefore do u still need to perform null coalesce here? 😉